source/_docs/free-https.md

```diff
@@ -171,6 +171,13 @@ No, please take a moment to fill out the [HTTPS survey](https://www.getfeedback.
 ### Is the CDN configurable? Do I get access to hit rates or other statistics?
 No, we pre-configured the CDN so you don’t have to hassle with configuration. We’ve optimized configuration for Drupal and WordPress sites. Hit rates or other statistics are not currently available.
 
+### Why does the Domains and HTTPS tool indicate action required even though I've updated DNS records?
+The action required message may be due to one of the following:
+
+  - **DNS changes can take some time to take effect:** Check the current state of DNS propagation from different parts of the world using this [free web tool](https://www.whatsmydns.net/).
+  -  **AAAA records not detected:**: Ensure you've added both AAAA records for bare domains (e.g. `example.com`) to route IPv6 traffic to your site. There are two AAAA records for improved uptime and reliability.
```

Comment:
> nsure you've added **both** AAAA records for bare domains

I suggest "both AAAA records for _the_ bare domain." As currently written, it sounds like **both** is referring to a second, unmentioned item.